rfc8446.xml

```diff
@@ -3115,8 +3119,7 @@ The server MUST generate a fresh value for each ticket it sends.</t>
   <t hangText='ticket:'>
   The value of the ticket to be used as the PSK identity.
 The ticket itself is an opaque label. It MAY be either a database lookup key
-or a self-encrypted and self-authenticated value.
-Section&nbsp;4 of <xref target="RFC5077"/> describes a recommended ticket construction mechanism.</t>
+or a self-encrypted and self-authenticated value.</t>
```

Comment:
IIUC we had some early email discussions about this text and all agreed to remove it.